modules/oe_theme_content_consultation/config/install/core.entity_view_display.node.oe_consultation.full.yml

```diff
@@ -0,0 +1,281 @@
+    - field.field.node.oe_consultation.oe_consultation_outcome
+    - field.field.node.oe_consultation.oe_consultation_outcome_files
+    - field.field.node.oe_consultation.oe_consultation_response_button
+    - field.field.node.oe_consultation.oe_consultation_target_audience
+    - field.field.node.oe_consultation.oe_departments
+    - field.field.node.oe_consultation.oe_subject
+    - field.field.node.oe_consultation.oe_summary
+    - field.field.node.oe_consultation.oe_teaser
+    - node.type.oe_consultation
+  module:
+    - datetime
+    - field_group
+    - link
+    - rdf_skos
+    - text
+    - user
+third_party_settings:
+  field_group:
+    group_page_contents:
+      children:
+        - group_details
+        - group_target_audience
+        - group_why_we_are_consulting
+        - group_responding_to_the_consultation
+        - group_consultation_outcome
+        - group_additional_information
+        - group_legal_notice
+        - group_contacts
+      parent_name: ''
+      weight: 0
+      format_type: oe_theme_helper_in_page_navigation
+      region: content
+      format_settings:
+        id: ''
+        classes: ''
+      label: 'Page contents'
+    group_details:
+      children:
+        - group_details_item
+      parent_name: group_page_contents
+      weight: 20
+      format_type: oe_theme_helper_in_page_navigation_item
+      region: content
+      format_settings:
+        id: ''
+        classes: ''
+      label: Details
+    group_details_item:
+      children:
+        - extra_field_oe_consultation_status
+        - oe_consultation_opening_date
+        - oe_consultation_deadline
+        - oe_departments
+      parent_name: group_details
+      weight: 11
+      format_type: oe_theme_helper_field_list_pattern
+      region: content
+      format_settings:
+        variant: horizontal
+        classes: ''
+        id: ''
+      label: ''
+    group_target_audience:
+      children:
+        - oe_consultation_target_audience
+      parent_name: group_page_contents
+      weight: 21
+      format_type: oe_theme_helper_in_page_navigation_item
+      region: content
+      format_settings:
+        id: ''
+        classes: ''
+      label: 'Target audience'
+    group_why_we_are_consulting:
+      children:
+        - oe_consultation_aim
+      parent_name: group_page_contents
+      weight: 22
+      format_type: oe_theme_helper_in_page_navigation_item
+      region: content
+      format_settings:
+        id: ''
+        classes: ''
+      label: 'Why we are consulting'
+    group_responding_to_the_consultation:
+      children:
+        - oe_consultation_guidelines
+        - extra_field_oe_consultation_closed_status_text
+        - oe_consultation_response_button
+      parent_name: group_page_contents
+      weight: 23
+      format_type: oe_theme_helper_in_page_navigation_item
+      region: content
+      format_settings:
+        id: ''
+        classes: ''
+      label: 'Responding to the consultation'
```

Comment:
Based on spec https://webgate.ec.europa.eu/fpfis/wikis/display/EWPP/Consultation+Content+Type+Specification+-+Version+1.0.0 is should be "Respond to the consultation".

Reply:
On the requirements page in the full view mock-up is "Responding to the consultation" in the full view fields labels is "Respond to the consultation". @ademarco which one we should keep?

Reply:
<img width="1051" alt="Screenshot 2021-01-25 at 11 52 17" src="https://user-images.githubusercontent.com/1183951/105682735-de0f0900-5f03-11eb-8573-902ea9f84a46.png">

I think it is designer's mistake